stellar-core/debian/rules

```diff
@@ -13,19 +13,18 @@ include /usr/share/dpkg/buildflags.mk
 # package maintainers to prepend CXXFLAGS
 export DEB_CXXFLAGS_MAINT_PREPEND  = -stdlib=libc++ -fno-omit-frame-pointer -isystem /usr/include/libcxxabi
 
-# set DEB_CONFIGURE_OPTS in your environment to override
```

Comment:
I know some community members use our Debian rules for their internal build systems but I am unaware if they rely on completely overriding the default configure options.

Apart from simplifying our internal build environments what is the reasoning for no longer being able to completely override the configure options passed to dh_auto_configure ?

Reply:
Simplification was the only reason really. But to be on the safe side, in case of community members using the option, I added it back.